ai_flow_plugins/job_plugins/flink/flink_version.py

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# The versions of Flink which include the DataSet API
+INCLUDE_DATASET_VERSIONS = ['1.11.0', '1.11.1', '1.11.2', '1.11.3', '1.11.4', '1.12.0', '1.12.1', '1.12.2', '1.12.3',
```

Comment:
What if the user installs pyflink==1.10?

Reply:
The minimum version of PyFlink AIFlow supports is 1.11.